### ISSUE TITLE
Export mapping.txt too

### ISSUE THREAD

**Issue by gaurav-fueled** [bitrise-io/steps-gradle-runner#6]
Please add functionality to export mapping.txt file too along with apk.

**Comment by gaurav-fueled** [comment]
For the moment I am using this script to get the path of mapping.txt file in an environment variable $BITRISE_MAPPING_PATH
``` 
 mapping_file_path=`find . -name "mapping.txt"`

if [[ "${mapping_file_path}" != "" ]] ; then
    echo 'Exporting output: $BITRISE_MAPPING_PATH =>' "${mapping_file_path}"
    envman add --key "BITRISE_MAPPING_PATH" --value "${mapping_file_path}"
else
    echo " (!) No mapping files matched the filters."
fi  ```

**Comment by godrei** [comment]
Solved in version 1.3.0, see: https://github.com/bitrise-io/steps-gradle-runner/releases/tag/1.3.0

**Comment by viktorbenei** [comment]
Still under automatic tests - should be in the Step Lib in about 10 minutes

**Comment by gaurav-fueled** [comment]
@viktorbenei Can we also let the users to specify the filters to select the correct mapping.txt on his own as is done for apks.

**Comment by viktorbenei** [comment]
Filters are now available in v1.3.1 @gaurav-fueled

**Comment by viktorbenei** [comment]
Once again, thanks for all the feedback & input @gaurav-fueled !
